src/hssm/hssm.py

```diff
@@ -164,10 +169,12 @@ def __init__(
         model_config: Config | None = None,
         loglik: str | PathLike | LogLikeFunc | pytensor.graph.Op | None = None,
         loglik_kind: LoglikKind | None = None,
+        hierarchical: bool = True,
```

Comment:
as metioned in high level comment, let's prob set this to false if print statements don't work. (However didn't understand why this breaks print statements)